UnitsNet/UnitDefinitions/Magnetization.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "Magnetization",
+  "BaseUnit": "AmperePerMeter",
+  "XmlDoc": "In classical electromagnetism, magnetization is the vector field that expresses the density of permanent or induced magnetic dipole moments in a magnetic material.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Magnetization",
+  "Units": [
```

Comment:
Any prefix units?
Add any of these units?
http://www.hkd.co.jp/english/unit_conv/